packages/bitcore-node/src/models/block.ts

```diff
@@ -23,6 +23,7 @@ export type IBlock = {
   bits: number;
   reward: number;
   processed: boolean;
+  raw: string;
```

Comment:
This `raw` value should be part of the `_apiTransform` return object so that the API actually returns it.